frontend/src/chrome/TopLevelLayout.tsx

```diff
@@ -0,0 +1,18 @@
+// for non-dataset views, contains Navbar and full-height content area with
+//overflow-y-scroll
+
+import React from 'react'
+import NavBar from '../features/navbar/NavBar'
+
+interface TopLevelLayoutProps {}
+
+const TopLevelLayout: React.FunctionComponent<TopLevelLayoutProps> = ({ children }) => (
```

Comment:
Use `FC` instead of `FunctionComponent`
```suggestion
const TopLevelLayout: React.FC<TopLevelLayoutProps> = ({ children }) => (
```